File: src/main/java/com/lord/small_box/dtos/ReceiptDto.java
package com.lord.small_box.dtos;

import java.util.ArrayList;
import java.util.Calendar;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class ReceiptDto {
	
	private String payment_type;
	
	private ArrayList<String> supplier_city;
	
	private String supplier_address;
	
	private ArrayList<String> line_item;
	
	private String total_price;
	
	private Calendar receipt_date;
	
	private String receipt_code;
	
	private ArrayList<String> supplier_name;

}

File: front/small-box/src/app/models/receiptDto.ts
export class ReceiptDto {
    
    payment_type!: string;

    supplier_city!: Array<string>;

    supplier_address!: string;

    line_item!: Array<string>;

    total_price!: string;

    receipt_date!: string;

    receipt_code!: string;

    supplier_name!: Array<string>;
}